apps/docs/src/theme/ReactLiveScope/index.tsx

```diff
@@ -4,7 +4,7 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import '@faststore/styles'
+// import '../../../../../packages/styles'
```

Comment:
hmm, do we need to change this file?

Reply:
in the future we should remove this import, but let's keep it, it was my bad I was testing and forgot to adjust it :)